sheetstorm/middleware/force_password_change.py

```diff
@@ -0,0 +1,25 @@
+from typing import Callable
+
+from django.http import HttpRequest
+from django.http import HttpResponse
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+
+from users.models import CustomUser
+
+
+class ForcePasswordChangeMiddleware:
```

Comment:
`ForcePasswordChangeMiddleware` -> `PasswordChangeMiddleware`. Class names should be more general. If it would be method/function it would be good :)

Reply:
Done.